ui/web/jquery.jqplot/MIT-LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+Title: MIT License
```

Comment:
Time came to start using `npm` (https://www.npmjs.com/, https://habr.com/ru/post/243335/).

Currently we just loaded `jquery` and `jqplot` libraries to folders and pushed them to the repository. This way was used during long time but now it's more popular to use `npm`.

To use `npm` we need:
1. Install `npm` to our local environment (load it from https://nodejs.org/en/download/);
1. Create `package.json` file in `work_estimation/ui/web` folder with next content:

```
{
  "devDependencies": {
    "jquery": "3.3.1",
    "jqplot": "1.0.9"
  }
}
```

2. Run next command  in `work_estimation/ui/web` folder: `npm install`
3. It will loads our libraries (`jquery` and `jqplot`) to `work_estimation/ui/web/node_modules`
4. Change imports in index.html file to next ones:

```
    <script language="javascript" type="text/javascript" src="node_modules/jquery/dist/jquery.min.js"></script>
    <script language="javascript" type="text/javascript" src="node_modules/jqplot/jquery.jqplot.min.js"></script>
    <link rel="stylesheet" type="text/css" href="node_modules/jqplot/jquery.jqplot.min.css" />
```

5. Check that everything works as it was before;
6. Remove folders `work_estimation/ui/web/jquery.jqplot` and `work_estimation/ui/web/lib/jquery`;
7. Add .gitignor file to folder `work_estimation/ui/web` and ignore `work_estimation/ui/web/node_modules` folder (it means that `node_modules` should be not committed and pushed to repository);
8. Commit and push next changes:
- removing of `work_estimation/ui/web/jquery.jqplot` and `work_estimation/ui/web/lib/jquery` folders;
- new package.json and package-lock.json (this file is generated during `npm install` command) files;
- changes in index.html file.


So in such way our libraries are not present in our repository. What is better since they don't add mess to the repository.

Reply:
All recommended changes have been made.

Reply:
Awesome! Thanks